.github/workflows/static-checks.yml

```diff
@@ -12,6 +12,8 @@ on:
       - 'internal/**'
       - 'orchestrion/**'
       - 'scripts/**'
+  schedule: # nightly
+    - cron: "0 0 * * *"
```

Comment:
This pipeline as nightly shouldn't deliver much value. This one is to block PRs from merging because they don't comply with our standards.

Reply:
done in [30fd573](https://github.com/DataDog/dd-trace-go/pull/4202/commits/30fd5736d562aae736fd646758e28e40367cf87d)